### ISSUE TITLE
Peer Review (jy478)

### ISSUE THREAD

**Issue by JohnYu0510** [erp78/5741Project-EarningsCallsStockPrices#15]
This project attempts to predict the overnight stock returns with the financial and trading data of firms. This is an interesting topic and can provide insights on how fundamental information of the firm can have an impact on its stock performance. They modeled the prediction task with a regression model and a classification model. Different training methods were used, and hyperparameters were tuned to avoid overfitting.

Things I like:
1.	A good visualization of data. In particular, I was impressed with figure 4 where the realized and estimated EPS were shown in a scatter plot. It directly illustrated why earnings surprise can determine the overnight stock returns.
2.	The modeling approach was illustrated clearly. I like the timeline graph which was very informative and helpful.
3.	Various loss functions, regularization terms, and error metrics were used to train and evaluate the model. The conclusion generally makes sense. 

Potential improvements:
1.	Nonlinearity: When constructing features like the P/E ratio, you will find that the lower P/E of a firm doesn’t indicate that it tends to be undervalued (since earnings can be negative). For example, a firm with a high stock price but negative EPS should be overvalued, but its P/E ratio is likely to be small (<0). So maybe some nonnegative ratios like P/B can work better.
2.	Scaling: Many features used to predict the overnight return have high correlations because they are selected from the original financial reports and all of them should be strongly correlated with the size of the firm. So in the feature engineering step, it’s better to divide the financial indicators by total assets or market caps.
3.	Return rates: The return of a single stock can be affected by the markets or economic conditions. For example, if there is some very good news happening in the markets, most of the stocks can go upward and it will be hard to estimate the idiosyncratic effect of the earnings call. To adjust for this, you can set the output variable as the return of this stock minus the average return of stocks in the same industry, so that you can better evaluate the idiosyncratic effect of the earnings call.
4.	More features: In this model, it turned out that the only feature that can affect the stock open prices significantly is earnings surprise. More useful features can be added to predict the returns. For example, you can use the stock return prior to the earnings call as a feature, since it can capture informed trading prior to earnings announcements.

Overall, this is an excellent and well-thought-out project. Good job!
